### DISCUSSION TITLE
Email Login Success... and nothing...

### DISCUSSION THREAD

**Discussion by SunSparc** [vercel/community#1065] [Help] [answered]
Is this a common problem?

- visit `https://vercel.com/login`
- "Continue with Email"
- check email
- click link
- the following page loads:
```
Email Login Success
Your email address was successfully authenticated.
You will now be redirected to the dashboard.
```
And stays there. Forever. I tried the process in three different browsers with the same result.

However, if I visit `https://vercel.com/accountname`, it takes me though the login process and allows me to proceed. So I cannot login from the Vercel home screen by clicking "Login" and then "Continue with Email". But as long as I know the URL of my dashboard, I can login just fine. Seems like a bug.

**Comment by swarnava** [comment, marked as answer]
Are you initiating login email and opening verification link on different tab? if you close the original tab, it won't work

**Comment by SunSparc** [comment]
I copy the link from the email and paste it into the same browser/window/tab that I initiated in. I just tried it using a different browser/window/tab, and it did work. Interesting user experience. I have not come across this process before.

**Reply by swarnava** [reply]
Thank you for confirming!

**Comment by gregg-cbs** [comment]
"Copy link from email and paste it into the same tab" - lol what kind of user experience is this. Been staring at my page for 10mins waiting for something to happen.
![image](https://user-images.githubusercontent.com/57114335/234011187-b542adba-506a-4e6e-9c72-42b0e6541ca5.png)

**Reply by amyegan** [reply]
You should not paste the address into the same tab. You should leave the original tab open, and either click to open the link from your email or copy and paste into a **new tab** to complete login.
